src/network/clientpackethandler.cpp

```diff
@@ -132,12 +132,10 @@ void Client::handleCommand_AuthAccept(NetworkPacket* pkt)
 					<< m_recommended_send_interval<<std::endl;
 
 	// Reply to server
-	std::string lang = gettext("LANG_CODE");
-	if (lang == "LANG_CODE")
-		lang = "";
+	const std::vector<std::string> &locale = getLanguageCodes();
+	NetworkPacket resp_pkt(TOSERVER_INIT2, 0);
+	resp_pkt << locale;
```

Comment:
network breakage

Reply:
Indeed. This PR was created during the unstable network phase. Will have to add guards around this code.